languages/ruby/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,24 @@
+There are two different types of numbers in Ruby:
```

Comment:
```suggestion
Two common types of numbers in Ruby:
```

Reply:
Mostly because, though I intuitively agree that it may be possible these are the two most common, I have not measured that to find out.

Reply:
> I have not measured that to find out

It seems like a real stretch to think of any scenario where rational or complex numbers are more common than integers and floats, but I also don't have any data to back that up.

Reply:
I could think of no scenario where float and integer wouldn't outweigh the "rational **or** complex", but I can think of some where it may be comparable or perhaps exceeded even, if we use **and** instead.